src/components/atoms/button/components/button.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Link } from 'gatsby';
+
+export const Button = ({
+  onClick,
+  href,
+  condition,
+  text,
+  target = 'button',
+  ...otherProps
+}) => {
+  if ((onClick && href) || (!onClick && !href)) return null;
```

Comment:
Not sure that we shouldn't render the button if both an `href` and an `onClick` are present, it wouldn't be semantically incorrect to have an anchor tag with an `onClick` function as well as an `href`.

Reply:
If there are both, there's the question about what the button should do, and the order

Reply:
imo if there's an `onClick` but no `href`, render a button,
if there's an `href` render a Link (regardless of whether there's an `onClick`),
and if there's neither, then null.